### DISCUSSION TITLE
How to add adobe signature status image (Question mark and green tick)

### DISCUSSION THREAD

**Discussion by Mallinath-debug** [MatthiasValvekens/pyHanko#166] [Q&A] [answered]
How to add adobe signature status image like  Question mark and green tick

**Comment by MatthiasValvekens** [comment, marked as answer]
Hi @Mallinath-debug,

Are you referring to a status indicator shown on the page as part of the visual appearance of the signature? Because that "feature" of Acrobat was deprecated (for _very_ good reasons; it's inherently insecure!) almost 20 years ago, and removed from the PDF specification altogether. In fact, it was never even part of any ISO PDF standard. Since pyHanko places a heavy emphasis on standards-based PDF, the old Acrobat appearance layer system is not and will never be supported.

You should train your users to look at the validation status report in the signature panel instead.

EDIT: If I misunderstood your question, please clarify. :)

**Comment by Mallinath-debug** [comment]
thank you
